### PR TITLE
Code Improvements

### PR DESCRIPTION
**Pull request by darxriggs**
After cleaning up the pom.xml I now prepared some commits to improve (just some aspects) the Java code.

### CODE REVIEW

**Comment by darxriggs** [comment]
Hi @jakub-bochenski, could you have a look at this too?

**Comment by jakub-bochenski** [comment]
Will try to do this tomorrow

File: src/main/java/jenkins/plugins/logstash/persistence/LogstashIndexerDao.java [hunk 1]
```diff
@@ -38,24 +37,24 @@
...
 	RFC5424,
 	RFC3164
   }
 
-  static enum SyslogProtocol {
+  enum SyslogProtocol {
 	UDP
   }
 
-  public String getDescription();
```
Comment (jakub-bochenski) [inline src/main/java/jenkins/plugins/logstash/persistence/LogstashIndexerDao.java]: you missed the extra semicolon here :)

File: src/main/java/jenkins/plugins/logstash/persistence/HostBasedLogstashIndexerDao.java [hunk 2]
```diff
@@ -48,27 +42,11 @@ public HostBasedLogstashIndexerDao(String host, int port) {
     }
   }
 
-  @Override
-  public JSONObject buildPayload(BuildData buildData, String jenkinsUrl, List<String> logLines) {
```
Comment (jakub-bochenski) [inline src/main/java/jenkins/plugins/logstash/persistence/HostBasedLogstashIndexerDao.java]: I'm surprised that this method wasn't called, let me double check

Comment (jakub-bochenski) [reply src/main/java/jenkins/plugins/logstash/persistence/HostBasedLogstashIndexerDao.java]: OK, it's the same as the super method

**Pull request merged**